dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
+
+import java.sql.SQLException;
+
+import dbfit.fixture.StatementExecution;
+
+public interface DbStoredProcedureCall {
+
+    String getName();
+
+    boolean isFunction();
+
+    String toSqlString();
```

Comment:
> @javornikolov any thoughts on, perhaps, removing the DbStoredProcedureCall interface and letting buildPreparedStatement of DbObject create the appropriate statement executor?

@MMatten, I took also a look at #463 which is touching related parts. Looks like what's enough for `DbObject` is quite simple (just a method to execute it and ability to release resources).

Maybe indeed a separate `DbStoredProcedureCall` interface would be redundant (something simpler like `DbCommand` should be enough).